rfcs/0000-data-structures/0000-data-structures.md

```diff
@@ -0,0 +1,265 @@
+| `args`        | [Bytes]    | **An array of arguments as the script input.** The arguments here are imported into the CKB-VM instance as input arguments for the scripts. Note that for lock scripts, the corresponding CellInput would have another args field which is appended to the array here to form the real input arguments. |
+
+
+
+More information about Script can be [here](https://github.com/nervosnetwork/ckb-demo-ruby-sdk/blob/master/docs/how-to-write-contracts.md#script-model).
+
+Also you can find how the `Script` structure is implemented from [these codes](https://github.com/nervosnetwork/ckb/blob/17b48a0777753a91691146b3817758c02db6f415/core/src/script.rs#L13-L53)
+
+
+
+## Transaction
+
+### Example
+
+```json
+{
+    "deps": [],
+    "hash": "0x4707810253259258e3091934dc5b543403c10cc899859e077fe26067f8d52dc0",
+    "inputs": [
+      {
+        "previous_output": {
+          "hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+          "index": 4294967295
+        },
+        "args": []
+      }
+    ],
+    "outputs": [
+      {
+        "capacity": 5000000,
+        "data": "0x",
+        "lock": {
+          "args": [],
+          "binary_hash": "0xa58a960b28d6e283546e38740e80142da94f88e88d5114d8dc91312b8da4765a",
+          "version": 0
+        }
+        "type": null
+      }
+    ],
+    "embeds": [
+      "0x0102030405"
+    ],
+    "version": 0
+}
+```
+
+### Description
+
+#### Transaction
+
+| Name              | Type                           | Description                                                  |
+| ----------------- | ------------------------------ | ------------------------------------------------------------ |
+| `version`         | uint32                         | **The version of the transaction.** It‘s used to distinguish transactions when there's a fork happened to the blockchain system. |
+| `hash`            | H256(Hash)                     | **The hash of the transaction.** This also serve as the identifier of the transaction. |
+| `deps`            | [`outpoint`]                   | **An array of `outpoint` that point to the cells that are dependencies of this transaction.** Only live cells can be listed here. The cells listed are read-only. |
+| `inputs`          | [{`previsou_output` , `args`}] | **An array of {`previous_output`, `args`}.** |
+| `embeds`          | [Bytes]                        | **An array of one-time used RISC-V binaries that can be referenced by `scripts` included in the transaction. ** |
```

Comment:
> **An array of one-time used RISC-V binaries that can be referenced by `scripts` included in the transaction. ** 

* The bold gramma seems not functioning. Delete the space before the latter **

->  **An array of one-time used RISC-V binaries that can be referenced by `scripts` included in the transaction.**